### PR TITLE
Tip button

### LINKED ISSUE #125: Add tip button to the chat
Use that util to check if user have chain name or account number and render [tip button](https://github.com/aeternity/jitsi-meet/blob/tip_button/react/features/conference/components/web/TipButton.js) near the user 
https://github.com/aeternity/jitsi-meet/blob/4c08c855909912599541643a38478d261b520ffe/react/features/aeternity/utils.js#L1-L9

### PR DESCRIPTION
**Pull request by shapkarin**

### CODE REVIEW

File: react/features/prejoin/components/Prejoin.js [hunk 1]
```diff
@@ -308,6 +309,14 @@ class Prejoin extends Component<Props, State> {
                                     type = 'secondary'>
                                     { 'Login with Superhero' }
                                 </ActionButton>}
+
+                                <div>
+                                    <TipButton
+                                        account = 'ak_21vaVFzQ98f4Kf9RoBSZDSr9zaiHfrni8PPDYMamuandeSogMA'
```
Comment (Chima1707) [inline react/features/prejoin/components/Prejoin.js]: Are we hard coding an AK address?

File: react/features/aeternity/components/TipButton.js [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+                {this.props.hasWallet ? <>
+                    <button onClick = { this._onToggleTooltip }>Tip</button>
+                    {isOpen && (
+                        <div>
+                            <input
+                                onChange = { this._onChangeValue }
+                                type = 'text'
+                                value = { this.state.value } />
+                            <button onClick = { this._onSendTip }>Send</button>
+                            <div>Ошибка если есть: {!showLoading && error && error}</div>
```
Comment (Chima1707) [inline react/features/aeternity/components/TipButton.js]: This is Russian

File: react/features/aeternity/components/TipButton.js [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+     *
+     * @param {Object} event - Contains new value.
+     * @returns {void}
+     */
+    _onChangeValue({ target: { value } }) {
+        this.setState({ value });
+    }
+
+    /**
+     * WIP.
```
Comment (Chima1707) [inline react/features/aeternity/components/TipButton.js:159]: Please can you remove the WIP documentation in this place and other places

**Review by Chima1707 — CHANGES_REQUESTED** [review]

File: react/features/prejoin/components/Prejoin.js (hunk 1 above)
Comment (shapkarin) [reply react/features/prejoin/components/Prejoin.js]: it's for the test. and I open PR just to see the diff

File: react/features/aeternity/components/TipButton.js (hunk 3 above)
Comment (shapkarin) [reply react/features/aeternity/components/TipButton.js:159]: OK

File: react/features/prejoin/components/Prejoin.js (hunk 1 above)
Comment (Chima1707) [reply react/features/prejoin/components/Prejoin.js]: Ok. got it.

**Review by Chima1707 — APPROVED** [review]

**Comment by Chima1707** [comment]
@mike-gusiev @shapkarin Nice job guys!

**Review by DeeJayElly — APPROVED** [review]

**Pull request merged**